specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -526,6 +526,10 @@
             "shareDeleteRetentionPolicy": {
               "$ref": "./common.json#/definitions/DeleteRetentionPolicy",
               "description": "The file service properties for share soft delete."
+            },
+            "protocolSettings": {
```

Comment:
Please add sample for it.
Why the bool value in such deep place? Will we add more properties in the Protocalsetting/SmbSetting/Multichannel in the near feature. we need this information in decide PSH design.

Reply:
It is to match the design of the data plane. From what I heard, there will be new protocol setting and new SMB setting in the future.